scripts/test

```diff
@@ -15,15 +15,12 @@ get_es_stats() {
 }
 
 validate_json() {
-  echo "Validating gp-data-merged.json"
-  jq '. | empty' tmp/data/input/gp-data-merged.json
-  echo "Validation exit code: $?"
-  exit_status=$(($? + exit_status))
-
-  echo "Validating mapping.json"
-  jq '. | empty' tmp/data/mapping/mapping.json
-  echo "Validation exit code: $?"
-  exit_status=$(($? + exit_status))
+  FILE=$1
+  echo "Validating $1"
```

Comment:
Sorry, was rushing to get my train. More haste, less speed.

Looked through the code and there are no `$1`'s anymore.